rules_building_block/execution_github_new_application_interaction_for_pat.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/10/11"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects a new application interaction event for a GitHub PAT not seen in the last 30 days.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "New Application Interaction For GitHub PAT"
+risk_score = 21
+rule_id = "9dc8ffb9-90ff-4fd1-8d25-1823eb347dad"
+severity = "low"
+tags = ["Domain: Cloud",
+        "Use Case: Threat Detection",
+        "Tactic: Execution",
+        "Rule Type: BBR",
+        "Data Source: Github"
+        ]
+timestamp_override = "event.ingested"
+type = "new_terms"
+building_block_type = "default"
+
+query = '''
+event.dataset: "github.audit" and event.category : "configuration" and 
+github.integration:* and github.hashed_token:* 
+and github.programmatic_access_type : ("OAuth access token" or "Fine-grained personal access token")
+'''
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1648"
+name = "Serverless Execution"
+reference = "https://attack.mitre.org/techniques/T1648/"
+
+
+[rule.threat.tactic]
+id = "TA0002"
+name = "Execution"
+reference = "https://attack.mitre.org/tactics/TA0002/"
+
+
+[rule.new_terms]
+field = "new_terms_fields"
+value = ["github.integration", "github.hashed_token"]
+
+[[rule.new_terms.history_window_start]]
+field = "history_window_start"
+value = "now-30d"
```

Comment:
Maybe reduce this to 14d to increase rule performance. In case it ends up noisy, it can always be bumped up. I would suggest the same for the others (and ofc reflect this in the rule descriptions).